get_id.js

```diff
@@ -1,9 +1,10 @@
-let local = JSON.parse(localStorage.USER);
-let token1 = local["token"]
+
 let r
 let resp
 
-let Url = "http://164.68.125.44:40895/table/items/get?limit=1&lang=1&id=2"
+let Url = "http://164.68.125.44:34351/table/items/get?limit=1&lang=1&id=2"

```

Comment:
must be parametrizase:
 not http://164.68.125.44:34351/ - name of host
 instead id=2 - id=&{id}

Reply:
++